docs/rundown.rst

```diff
@@ -228,3 +307,17 @@ stable:
      - Signing process failure
      - * The signer's private key was not found in the GPG keyring
        * The wrong GnuPG home directory or keyring file was specified
+
+Other Helpful Commands:
+======================
+``$ time ansible-sign project gpg-sign .``
+This command shows how long validation took.
```

Comment:
I don't know if this will be too useful to people. I ran it on a repo with ~4300 files and took under 0.4s for both signing and verifying. I think generally this shouldn't be an issue.